packages/moonstone/Slider/utils.js

```diff
@@ -29,7 +29,7 @@ const isNotMax = (ev, {value, max}) => {
 	return value !== max;
 };
 
-const isNotMin = (ev, {value, min}) => {
+const isNotMin = (ev, {min, value = min}) => {
```

Comment:
Is this not a failure that a unit test would catch?

Reply:
Good call! I added a couple tests for an undefined value.